apps/req.c

```diff
@@ -127,6 +127,9 @@ const OPTIONS req_options[] = {
      "Print the subject of the output request or cert"},
     {"multivalue-rdn", OPT_MULTIVALUE_RDN, '-',
      "Deprecated; multi-valued RDNs support is always on."},
+    {"not_before", OPT_NOT_BEFORE, 's', "Cert notBefore, YYMMDDHHMMSSZ"},
```

Comment:
Please make these texts a little more verbose, e.g.
`[YY]YYMMDDHHMMSSZ value to use for the notBefore certificate field`